src/applications/vaos/components/ConfirmedAppointmentListItem.jsx

```diff
@@ -97,8 +98,37 @@ export default function ConfirmedAppointmentListItem({
         )}
       </div>
 
+      {!allowCancel &&
+        !canceled && (
+          <div className="vads-u-margin-top--2">
+            <button
+              onClick={() => exportICSFile(appointment, facility)}
+              aria-label="App appointment to calendar"
+              className="vaos-appts__cancel-btn va-button-link vads-u-margin--0 vads-u-flex--0"
+            >
+              Add to calendar
+              <span className="sr-only">
```

Comment:
I don't think we need the sr only text here if we also have an `aria-label`. I would change `aria-label` to be `Add to calendar on {getAppointmentDate(appointment)}` and leave this button text as "Add to calendar".

Reply:
Done

Reply:
Looks like the sr-only filters the anchor text displayed.  Without it, the link displays, "Add to calendar on November 7, 2020" which doesn't match the screenshots. If anything, the aria-label should be removed and allow the screen reader to just read the anchor text.

Reply:
Ah, I meant that you could remove the date part from the link text. So the link should read "Add to calendar" and then aria-label would be "Add to calendar on November 7, 2020"

That keeps the link text short for sighted users, who can tell visually which date the link is associated with, but lets screen reader users get some extra context. We do a similar thing with the Edit links on the review page.

Reply:
Done